app-admin/profile-cleaner/profile-cleaner-2.36.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Vacuum and reindex browser sqlite databases"
+HOMEPAGE="https://github.com/graysky2/profile-cleaner"
+SRC_URI="${HOMEPAGE}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
Please don't use HOMEPAGE like this. It is multi-valued by design, and this will break hard if someone adds another homepage there.